pkg/adapter/containers.go

```diff
@@ -215,9 +236,16 @@ func (r *LocalRuntime) RemoveContainers(ctx context.Context, cli *cliconfig.RmVa
 		pool.Add(shared.Job{
 			ID: c.ID(),
 			Fn: func() error {
-				err := r.RemoveContainer(ctx, c, cli.Force, cli.Volumes)
+				err := r.RemoveContainer(ctx, c, force, cli.Volumes)
 				if err != nil {
-					logrus.Debugf("Failed to remove container %s: %s", c.ID(), err.Error())
+					if evict {
+						err = r.EvictContainer(ctx, []byte(c.ID()), cli.Volumes)
+						if err != nil {
+							logrus.Debugf("Failed to evict container %s: %s", c.ID(), err.Error())
```

Comment:
We're telling the user that we "failed to evict" the container, but no other documentation that I see that the user would come across.  I think "evict" happens when you do `-ff`.  We probably ought to add the word 'evict' to that new verbiage for `-ff`.

Reply:
I think you've my concern covered on this now with the other changes.  I'm going to resolve this.